@ngrx/data 8.5x breaking change

Looks like there's an issue when upgrading to @ngrx/data 8.5.2 

```
ERROR in ../../../node_modules/ngrx-etc/correlate/correlate-action.d.ts:2:23 - error TS2305: Module '"/node_modules/@ngrx/store/src/models"' has no exported member 'DisallowTypeProperty'.

2 import { TypedAction, **DisallowTypeProperty**, FunctionWithParametersType } from '@ngrx/store/src/models';
                        ~~~~~~~~~~~~~~~~~~~~

```

I've downgraded back to 8.4x for now. 
Let me know if i can help.

--- a/package.json
+++ b/package.json
@@ -57,16 +57,16 @@
   },
   "peerDependencies": {
     "@angular/core": ">=8.0.0-rc.1",
-    "@ngrx/store": ">=8.0.0",
-    "@ngrx/effects": ">=8.0.0"
+    "@ngrx/store": ">=8.5.0",
+    "@ngrx/effects": ">=8.5.0"
   },
   "dependencies": {
     "immer": "^3.1.1"
   },
   "devDependencies": {
     "@angular/core": "^8.0.0-rc.1",
-    "@ngrx/store": "^8.0.0",
-    "@ngrx/effects": "^8.0.0",
+    "@ngrx/store": "^8.5.0",
+    "@ngrx/effects": "^8.5.0",
     "@types/jest": "^24.0.11",
     "@typescript-eslint/eslint-plugin": "^1.7.0",
     "@typescript-eslint/parser": "^1.7.0",

--- a/src/correlate/correlate-action.ts
+++ b/src/correlate/correlate-action.ts
@@ -1,5 +1,5 @@
 import { Creator, createAction, ActionCreator } from '@ngrx/store'
-import { TypedAction, DisallowTypeProperty, FunctionWithParametersType } from '@ngrx/store/src/models'
+import { TypedAction, FunctionWithParametersType, DisallowArraysAndTypeProperty } from '@ngrx/store/src/models'
 
 export interface TypedActionWithCorrelation<T extends string> extends TypedAction<T> {
   __correlationId: string
@@ -17,7 +17,7 @@ export function createActionWithCorrelation<T extends string, P extends object>(
 ): ActionCreator<T, (props: P) => P & TypedActionWithCorrelation<T>>
 export function createActionWithCorrelation<T extends string, P extends any[], R extends object>(
   type: T,
-  creator: Creator<P, DisallowTypeProperty<R>>,
+  creator: Creator<P, DisallowArraysAndTypeProperty<R>>,
 ): FunctionWithParametersType<P, R & TypedActionWithCorrelation<T>> & TypedActionWithCorrelation<T>
 export function createActionWithCorrelation<T extends string, P extends any[], R extends object>(...args: any[]): any {
   return correlateAction((createAction as any)(...args))

--- a/src/mutable-on/mutable-on.ts
+++ b/src/mutable-on/mutable-on.ts
@@ -1,7 +1,5 @@
 import produce, { Draft } from 'immer'
-import { Action, ActionCreator } from '@ngrx/store'
-import { ActionType } from '@ngrx/store/src/models'
-import { On } from '@ngrx/store/src/reducer_creator'
+import { Action, ActionCreator, ActionType, On } from '@ngrx/store'
 
 export interface MutableOnReducer<S, C extends ActionCreator[], D = Draft<S>> {
   (state: D, action: ActionType<C[number]>): void

--- a/tests/mutable-on/mutable-on.spec.ts
+++ b/tests/mutable-on/mutable-on.spec.ts
@@ -2,7 +2,7 @@ import {
   createReducer,
   createAction,
   on,
-  ɵngrx_modules_store_store_z as createImmutabilityCheckMetaReducer,
+  ɵngrx_modules_store_store_bb as createImmutabilityCheckMetaReducer,
   props,
   ActionReducer,
   Action,

